Plugin/StoreLocatorBlockViewMapPlugin.php

```diff
@@ -58,28 +58,25 @@ public function __construct(
     }
 
     /**
-     * Add promotion in markers.
-     *
-     * @param \Smile\StoreLocator\Block\Search  $block  The block search
-     * @param $result                           $result List of markers
-     * @return  array
+     * * Add promotion in markers.
+     * @param \Smile\StoreLocator\Block\View\Map $block The block Map
+     * @param $result                            $result List of markers
+     * @return mixed
      * @throws \Magento\Framework\Exception\LocalizedException
      */
-    public function afterGetMarkers(\Smile\StoreLocator\Block\Search $block, $result)
+    public function afterGetMarkerData(\Smile\StoreLocator\Block\View\Map $block, $result)
     {
```

Comment:
I don't understand this plugin function because we don't have a function named getMarkerData() in Smile\StoreLocator\Block\View\Map > https://github.com/Smile-SA/magento2-module-store-locator/blob/master/Block/View/Map.php

Reply:
can you explain to me ?